lib/hpack/huffman.ex

```diff
@@ -24,543 +24,1054 @@ defmodule HPack.Huffman do
-  defp dec(<< 0x6e::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<83>>)
-  defp dec(<< 0x6f::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<84>>)
-  defp dec(<< 0x70::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<85>>)
-  defp dec(<< 0x71::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<86>>)
-  defp dec(<< 0x72::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<87>>)
-  defp dec(<< 0xfc::8, rest::bitstring >>, length, result), do: dec(rest, length - 8, result <> <<88>>)
-  defp dec(<< 0x73::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<89>>)
-  defp dec(<< 0xfd::8, rest::bitstring >>, length, result), do: dec(rest, length - 8, result <> <<90>>)
-  defp dec(<< 0x1ffb::13, rest::bitstring >>, length, result), do: dec(rest, length - 13, result <> <<91>>)
-  defp dec(<< 0x7fff0::19, rest::bitstring >>, length, result), do: dec(rest, length - 19, result <> <<92>>)
-  defp dec(<< 0x1ffc::13, rest::bitstring >>, length, result), do: dec(rest, length - 13, result <> <<93>>)
-  defp dec(<< 0x3ffc::14, rest::bitstring >>, length, result), do: dec(rest, length - 14, result <> <<94>>)
-  defp dec(<< 0x22::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<95>>)
-  defp dec(<< 0x7ffd::15, rest::bitstring >>, length, result), do: dec(rest, length - 15, result <> <<96>>)
-  defp dec(<< 0x3::5, rest::bitstring >>, length, result), do: dec(rest, length - 5, result <> <<97>>)
-  defp dec(<< 0x23::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<98>>)
-  defp dec(<< 0x4::5, rest::bitstring >>, length, result), do: dec(rest, length - 5, result <> <<99>>)
-  defp dec(<< 0x24::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<100>>)
-  defp dec(<< 0x5::5, rest::bitstring >>, length, result), do: dec(rest, length - 5, result <> <<101>>)
-  defp dec(<< 0x25::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<102>>)
-  defp dec(<< 0x26::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<103>>)
-  defp dec(<< 0x27::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<104>>)
-  defp dec(<< 0x6::5, rest::bitstring >>, length, result), do: dec(rest, length - 5, result <> <<105>>)
-  defp dec(<< 0x74::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<106>>)
-  defp dec(<< 0x75::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<107>>)
-  defp dec(<< 0x28::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<108>>)
-  defp dec(<< 0x29::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<109>>)
-  defp dec(<< 0x2a::6, rest::bitstring >>, length, result), do: de
```

Comment:
I disagree with this change. This new version is not readable anymore.

Reply:
Fixed in 3d8fdae